### ISSUE TITLE
Convert Circle of Fifths Component to Accessible Grid

### ISSUE THREAD

**Issue by TomChamberlainUK** [TomChamberlainUK/music#89]
# Convert Circle of Fifths Component to Accessible Grid

Set the roles within the circle of fifths to mimic a grid to maximise accessibility

## Description

Currently our `<CircleOfFifths />` component looks visibly suitable but fails to be accessible via screen readers.

We should apply accessible roles to mimic the behaviour of a grid, to make it both keyboard and screenreader navigable.

## Done Criteria

- [ ] SVG is an accessible grid
- [ ] Major, minor, and diminished chords are separate accessible grid rows
- [ ] Individual chords are accessible grid cells
- [ ] Circle of Fifths is screen reader accessible
- [ ] Circle of Fifths is keyboard accessible
